From 7ede0d4ce19e0ed14cd1cec81ba7df4e467f1eeb Mon Sep 17 00:00:00 2001
From: NIIBE Yutaka <gniibe@fsij.org>
Date: Mon, 21 Jun 2021 12:16:20 +0900
Subject: [PATCH] build: Fix --disable-threads.

* src/gen-lock-obj.sh: Fix generating '}}}'.

--

GnuPG-bug-id: 5495
Reported-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Fixes-commit: 1fb90a7da186ee2ee098a666f6f3a35bb1720e59
Signed-off-by: NIIBE Yutaka <gniibe@fsij.org>
[Retrieved from:
https://git.gnupg.org/cgi-bin/gitweb.cgi?p=libgpg-error.git;a=commit;h=220a427b4f997ef6af1b2d4e82ef1dc96e0cd6ff]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 src/gen-lock-obj.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gen-lock-obj.sh b/src/gen-lock-obj.sh
index 258eec6..014667f 100755
--- a/src/gen-lock-obj.sh
+++ b/src/gen-lock-obj.sh
@@ -121,10 +121,10 @@ while test "$i" -lt $ac_mtx_size; do
     fi
     i=$(( i + 1 ))
 done
+echo '}}}'
 fi
 
 cat <<'EOF'
-}}}
 ##
 ## Local Variables:
 ## mode: c
-- 
2.31.1

